packages/lwt-canceler/lwt-canceler.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://gitlab.com/nomadic-labs/lwt-canceler"
+bug-reports: "https://gitlab.com/nomadic-labs/lwt-canceler/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/lwt-canceler.git"
+license: "MIT"
+depends: [
+  "ocaml"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
according to the dune-project file.
Are you sure that lwt does not need lower bounds?